src/Publisher/Tapatalk/TopicPublisher.php

```diff
@@ -31,10 +31,15 @@ protected function createEntry(
         string $body
     ): void {
         $result = $client->topics()->createNewTopic($configuration['forumId'], $subject, $body);
-        $sticky = ($configuration['sticky'] ?? false);
+        $sticky = $configuration['sticky'] ?? false;
+        $close = $configuration['close'] ?? false;
```

Comment:
The brackets are required here by the code style conventions (`Squiz.Formatting.OperatorBracket`). Please readd and add it to the new line as well.